### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
-Diagram a data model clearly and explain what it's doing.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ALTER TABLE
+    "invoice_items" ADD PRIMARY KEY("id");
+CREATE TABLE "treatment_histories"(
+    "medical_histories_id" INTEGER NOT NULL,
+    "treatment_id" INTEGER NOT NULL
+);
+ALTER TABLE
+    "treatment_histories" ADD PRIMARY KEY("medical_histories_id");
+ALTER TABLE
+    "treatment_histories" ADD PRIMARY KEY("treatment_id");
```
Comment (danmainah) [inline schema_based_on_diagram.sql]: - [ ] Kindy check on your alter table commands, `medical_histories_id` and `treatment_id` are supposed to be foreign keys.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_invoice_id_foreign" FOREIGN KEY("invoice_id") REFERENCES "invoices"("id");
+ALTER TABLE
+    "medical_histories" ADD CONSTRAINT "medical_histories_patient_id_foreign" FOREIGN KEY("patient_id") REFERENCES "patients"("id");
+ALTER TABLE
+    "invoices" ADD CONSTRAINT "invoices_medical_history_id_foreign" FOREIGN KEY("medical_history_id") REFERENCES "medical_histories"("id");
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_treatement_id_foreign" FOREIGN KEY("treatement_id") REFERENCES "treatment"("id");
```
Comment (danmainah) [inline schema_based_on_diagram.sql:67]: - [ ] Kindly create indexes for all your foreign keys.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83 and @udeaghad  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @MICHAELMUNAVU83, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**